tests/integration/cqlengine/model/test_updates.py

```diff
@@ -278,24 +305,32 @@ def test_collection_with_default(self):
         @test_category object_mapper
         """
         sync_table(ModelWithDefaultCollection)
-        item = ModelWithDefaultCollection.create(id=1, mf={1: 1}, dummy=1).save()
-        self.assertEqual(ModelWithDefaultCollection.objects().all().get()._as_dict(),
-                         {'id': 1, 'dummy': 1, 'mf': {1: 1}})
 
-        item.update(mf={2: 2})
-        self.assertEqual(ModelWithDefaultCollection.objects().all().get()._as_dict(),
-                         {'id': 1, 'dummy': 1, 'mf': {2: 2}})
+        udt, udt_default = UDT(age=1, mf={6: 6}), UDT(age=1, mf={6: 6})
+
+        item = ModelWithDefaultCollection.create(id=1, mf={1: 1}, dummy=1, udt=udt, udt_default=udt_default).save()
+        self.assertEqual(ModelWithDefaultCollection.all().get()._as_dict(),
+                         {'id': 1, 'dummy': 1, 'mf': {1: 1}, "udt": udt, "udt_default": udt_default})
+
+        udt, udt_default = UDT(age=1, mf={5: 5}), UDT(age=1, mf={5: 5})
+        item.update(mf={2: 2}, udt=udt, udt_default=udt_default)
+        self.assertEqual(ModelWithDefaultCollection.all().get()._as_dict(),
+                         {'id': 1, 'dummy': 1, 'mf': {2: 2}, "udt": udt, "udt_default": udt_default})
 
-        item.update(mf=None)
-        self.assertEqual(ModelWithDefaultCollection.objects().all().get()._as_dict(),
-                         {'id': 1, 'dummy': 1, 'mf': {}})
+        udt, udt_default = UDT(age=1, mf=None), UDT(age=1, mf=None)
+        expected_udt, expected_udt_default = UDT(age=1, mf={}), UDT(age=1, mf={})
+        item.update(mf=None, udt=udt, udt_default=udt_default)
+        self.assertEqual(ModelWithDefaultCollection.all().get()._as_dict(),
+                         {'id': 1, 'dummy': 1, 'mf': {}, "udt": expected_udt, "udt_default": expected_udt_default})
 
+        udt_default = UDT(age=1, mf=None), UDT(age=1, mf={5:5})
         item = ModelWithDefaultCollection.create(id=2, dummy=2).save()
-        self.assertEqual(ModelWithDefaultCollection.objects().all().get(id=2)._as_dict(),
-                         {'id': 2, 'dummy': 2, 'mf': {2: 2}})
+        self.assertEqual(ModelWithDefaultCollection.all().get(id=2)._as_dict(),
+                         {'id': 2, 'dummy': 2, 'mf': {2: 2}, "udt": None, "udt_default": udt_default})
```

Comment:
Not sure to follow what we are testing here. udt_default is not used in the create()